### PR TITLE
ci: do not run lints on drafts to make reviewing easier

### PR DESCRIPTION
**Pull request by justinchuby**
image

### CODE REVIEW

**Review by gramalingam — APPROVED** [review]
Very helpful, thanks!

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -35,6 +36,8 @@ jobs:
   enforce-style:
     name: Enforce style
     runs-on: ubuntu-latest
+    # Do not run on drafts to make reviewing easier
+    if: github.event.pull_request.draft == false
```
Comment (justinchuby) [inline .github/workflows/lint.yaml:40]: Does this work in commits?

Comment (justinchuby) [reply .github/workflows/lint.yaml:40]: Let's just merge it and try.

**Review by titaiwangms — APPROVED** [review]

File: .github/workflows/lint.yaml (hunk 1 above)
Comment (justinchuby) [reply .github/workflows/lint.yaml:40]: Good: it works

**Comment by gramalingam** [comment]
A question: does this distinguish between CI failures that affect the browser view of the code from those that don't? I think that the important thing to disable is the first category, not necessarily the second.

**Comment by gramalingam** [comment]
Is there a difference between what is checked and reported by the messages logged in the browser view of code and what is checked by lintrunner etc. ?

**Comment by justinchuby** [comment]
They should be the same

**Comment by justinchuby** [comment]
Oh! I _could_ just skip the web reporting part of it in draft mode and retain the lintrunner run; is that what you would suggest?

**Comment by gramalingam** [comment]
Yes, I think that could be useful. It is not invasive to the reviewer, but can remind the PR creator if they have anything to fix.

**Comment by justinchuby** [comment]
SG!

**Pull request merged**